datafusion/tests/sql.rs

```diff
@@ -5999,3 +5999,75 @@ async fn test_expect_distinct() -> Result<()> {
     assert_batches_eq!(expected, &actual);
     Ok(())
 }
+
+#[tokio::test]
+async fn test_nulls_first_asc() -> Result<()> {
+    let mut ctx = ExecutionContext::new();
+    let sql = "SELECT * FROM (VALUES (1, 'one'), (2, 'two'), (null, 'three')) AS t (num,letter) ORDER BY num";
```

Comment:
Every time I construct a test case with a **value list**, I think of @Jimexist lol 😄

Reply:
yeah it is so handy